addons/binding/org.openhab.binding.mysensors/ESH-INF/thing/thing-types.xml

```diff
@@ -0,0 +1,1476 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<thing:thing-descriptions bindingId="mysensors" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+	xmlns:thing="http://eclipse.org/smarthome/schemas/thing-description/v1.0.0"
+	xsi:schemaLocation="http://eclipse.org/smarthome/schemas/thing-description/v1.0.0 http://eclipse.org/smarthome/schemas/thing-description-1.0.0.xsd">
+
+	<!-- GATEWAY -->
+
+	<bridge-type id="bridge-ser">
+		<label>MySensors Serial Gateway</label>
+		<description>MySensors Gateway connected via serial port
+		</description>
+		<config-description>
+			<parameter name="serialPort" type="text" required="true">
+				<label>Serial Port</label>
+				<description>The serial port the MySensors Gateway is connected to
+				</description>
+			</parameter>
+			<parameter name="sendDelay" type="integer" required="false" min="1" max="1000">
+				<label>Send Delay</label>
+				<description>The delay between messages send to the gateway 
+				</description>
+				<default>100</default>
+			</parameter>
+			<parameter name="baudRate" type="integer" required="false" min="300" max="250000">
+				<label>Baudrate</label>
+				<description>Baudrate for serial port
+                </description>

```

Comment:
Once remarked for this, file, happens many times: Please use tabs instead of spaces for indenting.